data/system_outputs/text-classification-custom-feature-example.json

```
@@ -0,0 +1,1035 @@
+{
+  "metadata": {
+    "custom_features": {
+      "example": {
+        "verb": {
+          "dtype": "string",
+          "description": "the verb in the example"
+        },
+        "subject": {
+          "dtype": "string",
+          "description": "the subject in the example"
+        },
+        "adverb": {
+          "dtype": "string",
+          "description": "the adverb in the example"
+        },
+        "LL_delta": {
+          "dtype": "float",
+          "description": "log likelihood / confidence"
+        }
```

The newest version of the repository requires to always specify "cls_name" for every feature.
```suggestion
        "verb": {
          "cls_name": "Value",
          "dtype": "string",
          "description": "the verb in the example"
        },
        "subject": {
          "cls_name": "Value",
          "dtype": "string",
          "description": "the subject in the example"
        },
        "adverb": {
          "cls_name": "Value",
          "dtype": "string",
          "description": "the adverb in the example"
        },
        "LL_delta": {
          "cls_name": "Value",
          "dtype": "float",
          "description": "log likelihood / confidence"
        }
```

---

Thanks for the correction. What does "cls_name" mean for each feature? Is it the same as feature name or is it always 'Value'?

---

"cls_name" is used to distinguish which type we have to map these objects to. If the feature is either number or string, it should be "Value".